app/models/book_suggestion.rb

```diff
@@ -0,0 +1,3 @@
+class BookSuggestion < ApplicationRecord
+  validates :author, :title, :link, presence: true
```

Comment:
Sure the book suggestion has no relationships? Doesn't it belong to a user? When you define a new model always check for relationships, because doing book_suggestion.user might be helpful in many contexts